web/src/main/java/org/wso2/testgrid/web/api/SSOService.java

```diff
@@ -65,7 +66,8 @@
       * @return Redirection to dashboard home page.
```

Comment:
Since this is now redirected to requesting page, shall we update the comment as well?